test/drivers/test_suites/syscall_exit_suite/open_by_handle_at_x.cpp

```diff
@@ -99,14 +99,15 @@ void do___open_by_handle_atX_success(int *open_by_handle_fd, int *dirfd, char *f
 		}
 	}
 
+#ifndef __loongarch64
```

Comment:
Can we add a small comment around these compilation guards? Eg: "`fstat` not available on loongarch64" is enough :) 

https://github.com/falcosecurity/libs/pull/1654#discussion_r1571705926

Reply:
Of course, this is what I should do, and I will add them one by one in other sections.

Reply:
Perhaps it would be even better to just check for `ifdef __NR_fstat` ? 
Sorry, i came up with that right now :laughing: 
Since it is self explaining, then, we can avoid adding the comment!